brambling/forms/orders.py

```diff
@@ -134,27 +134,19 @@ def __init__(self, *args, **kwargs):
 
                 owner = self.instance.person
                 self.initial.update({
-                    'ef_cause': self.filter(EnvironmentalFactor.objects.only('id'),
-                                            person_cause=owner),
-                    'ef_avoid': self.filter(EnvironmentalFactor.objects.only('id'),
-                                            person_avoid=owner),
+                    'ef_cause': owner.ef_cause.all(),
+                    'ef_avoid': owner.ef_avoid.all(),
```

Comment:
Does this mean that these fields will both be filled in with all possible options initially?